node/hubRouter.js

```diff
@@ -2,7 +2,8 @@ var q = require('q');
 var express = require('express');
 var hubRouter = express.Router();
 var config = require("./config");
-var OpenT2T = require("opent2t").OpenT2T;
+var opent2t = require("opent2t").OpenT2T;
+var OpenT2T = new opent2t();
```

Comment:
why aren't we following the same convention as in other places? Capital OpenT2T for class and opent2t for instance?